File: June/12-06-2024/Count numbers containing 4.py

class Solution:
    def countNumberswith4(self, n : int) -> int:
    # Your Code Start
        count = 0
        for i in range(1, n + 1):
            if '4' in str(i):
                count += 1
        return count
        
#Your COde End





#{ 
 # Driver Code Starts
if __name__ == "__main__":
    t = int(input())
    for _ in range(t):

        n = int(input())

        obj = Solution()
        res = obj.countNumberswith4(n)

        print(res)

# } Driver Code Ends

File: June/12-06-2024/Count numbers containing 4.cs
//{ Driver Code Starts
using System;

namespace DriverCode {
class GFG {
    static void Main(string[] args) {
        int testcases;
        testcases = Convert.ToInt32(Console.ReadLine());
        while (testcases-- > 0) {

            int n = Convert.ToInt32(Console.ReadLine().Trim());

            Solution obj = new Solution();
            int res = obj.countNumberswith4(n);
            Console.WriteLine(res);
        }
    }
}
}

// } Driver Code Ends





class Solution {
    public int countNumberswith4(int n) {
        //Your Code Start
        int count = 0;
        for (int i = 1; i <= n; i++) {
            if (i.ToString().Contains('4')) {
                count++;
            }
        }
        return count;
      //Your Code End
    }
}
